src/main/groovy/wooga/gradle/paket/unity/tasks/PaketUnityInstall.groovy

```diff
@@ -122,6 +126,17 @@ class PaketUnityInstall extends ConventionTask {
     PaketUnityInstall() {
         description = 'Copy paket dependencies into unity projects'
         group = PaketUnityPlugin.GROUP
+
+        this.doLast {
+            if (isPaketUpmPackageEnabled().get()) {
+                def references = new PaketUnityReferences(getReferencesFile())
+                def locks = new PaketLock(getLockFile())
+                def packages = locks.getAllDependencies(references.nugets).collect {
```

Comment:
one thing that we need to make sure is, that ONLY whatever was being installed in Assets/Paket.Unity3D gets installed in Packages/ too, and nothing more. So if there is any dependency of something referenced in the unity3d reference file but it would have NOT be installed in there, it should not be installed in Packages either. 

My concern is to remind you that paket actually only copies the .net dll variants that are defined/supported inside Paket.Unity3D.

Reply:
@pletoss this task is the exact same one the installs stuff in `Assets/Paket.Unity3D`. All I'm doing there is changing the installation directory and making sure that the package.json file is in the right place. So don't worry about this ;)